src/main/java/sirius/pasta/tagliatelle/compiler/TemplateCompiler.java

```diff
@@ -379,6 +383,8 @@ private void parseAttributes(TagHandler handler) {
             verifyAttributeNameAndType(handler, name, attributeType);
             if (attributeType == null) {
                 attributeType = String.class;
+            } else {
```

Comment:
Question: why an else? An attribute without a type just defaults to String, but its still an attribute or am I missing something?

Reply:
When there is no attributeType it already creates the error "Unknown attribute." That means, the attribute is not defined in the tag. Therefore i think the attribute is not needed in the Set.